lib/tests/streamlit/web/server/media_file_handler_test.py

```diff
@@ -80,5 +82,5 @@ def test_downloadable_file(self, _) -> None:
 
     def test_invalid_file(self) -> None:
         """Requests for invalid files fail with 404."""
-        rsp = self._fetch_file("invalid_media_file")
+        rsp = self.fetch("/media/invalid_media_file.mp4")
```

Comment:
Nit: Should use the `MOCK_ENDPOINT` string

Reply:
(Whoops, this uncovered a minor issue - now fixed - where MediaFileHandler was improperly returning a 5xx error code, instead of a 404.)